src/components/List/List.tsx

```diff
@@ -2,33 +2,92 @@ import * as React from 'react';
 import {ListItemType} from 'types/types';
 import Card from '../Card/Card';
 import {Spinner} from '../Spinner/Spinner';
-import {ListContainer} from './List.styled';
+import {ItemsPerPage, ListContainer, PageNumber, PageSelect, PaginationContainer} from './List.styled';
 
 interface ListProps {
     items: ListItemType[];
     hasNavigation?: boolean;
     isLoading: boolean;
+    isUser?: boolean;
+    itemsPerPage?: number;
+    usePagination?: boolean;
```

Comment:
Making pagination optional, because on the Teams page is useful, but on the team's overview page there isn't many team members to need pagination, so the component should have it as optional for when it's a good fit only